javascript-timers/timeout.js

```diff
@@ -0,0 +1,6 @@
+const $heading = document.querySelector('.message');
+function helloThere() {
+  $heading.textContent = 'Hello There';
+  clearTimeout(timer);
+}
+const timer = setTimeout(helloThere(), 2000);
```

Comment:
Make sure you're not calling `helloThere` before the timer starts.

Reply:
@hollyphung95 Please make sure you're passing `setTimeout` a reference to the `helloThere` function, not passing its return value.